Source/Game/Game/ECS/Systems/UpdatePhysics.cpp

```diff
@@ -40,14 +40,14 @@ namespace ECS::Systems
 		{
 			auto& transform = registry.get<ECS::Components::Transform>(entity);
 
-			JPH::BoxShapeSettings shapeSetting(JPH::Vec3(transform.scale.x * 0.5f, transform.scale.y * 0.5f, transform.scale.z * 0.5f));
+			JPH::BoxShapeSettings shapeSetting(JPH::Vec3(transform.GetScale().x * 0.5f, transform.GetScale().y * 0.5f, transform.GetScale().z * 0.5f));
```

Comment:
I think it makes sense to create a local variable above called scale instead of calling GetScale directly 3 times, same goes for position below (All cases where this happens)